core/build.gradle

```diff
@@ -92,6 +92,8 @@ dependencies {
         exclude(group: 'org.apache.commons', module: 'commons-compress')
     }
 
+    shaded "org.yaml:snakeyaml:1.21"
```

Comment:
Not a big deal, but we already have Jackson on the classpath. Did we consider using that instead of SnakeYAML?

Reply:
Jackson does not support YAML. There is YAML module for Jackson that uses SnakeYAML internally :)

Reply:
I wasn't aware that it used SnakeYAML! OK, makes sense!